Adapters/ControlAdapter.cs

```diff
@@ -42,6 +48,38 @@ public ControlAdapter(Control control)
             ArgChecker.AssertArgNotNull(control, "control");
 
             _control = control;
+
+            _control.PointerMoved += (sender, args) =>
+            {
+                var visualRoot = _control.GetVisualRoot();
+                if (visualRoot != null)
+                {
+                    _lastPosition = args.GetPosition(visualRoot);
+                }
+            };
+            
+            _control.PointerPressed += (sender, args) =>
+            {
+                _leftMouseButton = true;
+            };
+            
+            _control.PointerReleased += (sender, args) =>
+            {
+                switch (args.InitialPressMouseButton)
+                {
+                    case MouseButton.Left:
+                        _leftMouseButton = false;
+                        break;
+                    case MouseButton.Right:
+                        _rightMouseButton = false;
+                        break;
+                }
+            };
+            
+            _control.ContextRequested += (sender, args) =>
+            {
+                _rightMouseButton = true;
```

Comment:
ContextRequested will also be called when users opens a context with a platform depended hotkey. Is it expected?

Reply:
Seems reasonable, as it is equivalent, isn't it?

Reply:
Yes, as for me that's fine. Just wanted to confirm.